packages/react/src/components/TextArea/TextArea.js

```diff
@@ -184,6 +244,16 @@ TextArea.propTypes = {
    * Specify whether you want the light version of this control
    */
   light: PropTypes.bool,
+
+  /**
+   * Specify whether the character counter is shown
+   */
+  charCount: PropTypes.bool,
+
+  /**
+   * The maximum allowed input value length
+   */
+  maxLength: PropTypes.number,
```

Comment:
Need `onChange` prop, too, with "missing `onChange` in controlled mode" check.

Reply:
this should be addressed now